homeassistant/components/mqtt/climate.py

```diff
@@ -779,6 +855,19 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
 
         self.async_write_ha_state()
 
+    async def async_set_humidity(self, humidity: int) -> None:
+        """Set new target humidity."""
+
+        await self._set_climate_attribute(
```

Comment:
We should check for `ClimateEntityFeature.TARGET_HUMIDITY` to be set here and raise some error if this is not supported.